scripts/commands.py

```diff
@@ -1,28 +1,18 @@
 import browse
 import json
-from memory import PineconeMemory
+import memory as mem
```

Comment:
Question  - if Pinecone isn't being used, how does the program know which mem to use?

Reply:
Good question, Honestly I couldn't figure out why that worked and not PineconeMemory.

Reply:
There is no usage of `memory` here. The only command that needed `memory` was `memory.add`. Which seems to have been removed away from the prompt. 

There is very little need for `memory.add` command in the first place since we're storing to the embeddings vector database for every command.